index.html

```diff
@@ -296,4 +297,125 @@ <h2>About me</h2>
       crossorigin="anonymous"
     ></script>
   </body>
+
+  <footer id="contact">
+    <div class="footer-text">
+      <h2 class="title">Contact me</h2>
+      <p class="supporting-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+    </div>
+    <form
+      action="https://formspree.io/f/xwkyvenp"
+      method="get"
+      class="form"
+      id="form"
+    >
+      <div class="name">
+        <label for="name"></label>
+        <input
+          type="text"
+          name="fullName"
+          id="name"
+          placeholder="Full name"
+          required
+          minlength="2"
+          maxlength="20"
+        />
+      </div>
+      <div class="email">
+        <label for="mail"></label>
+        <input
+          type="email"
+          name="mail"
+          id="mail"
+          placeholder="Email"
+          required
+        />
+      </div>
+      <div class="message">
+        <label for="message"></label>
+        <textarea
+          name="message"
+          id="message"
+          placeholder="Enter text here"
+          required
+          maxlength="300"
```

Comment:
[ ] kindlly change the maxlength to 500 , Message field should accept up to 500 chars